### PR TITLE
Adds loot crates.

### PR DESCRIPTION
**Pull request by DrSnips**
This PR was sponsored by loot crate.
Loot crate is a monthly subscription for $30.00 to get whacky nerd stuff in the mail straight to your door every month. Sign up for one today with the promo code "FUCKYOUDUNY" for 30% off your first loot crate subscription. They even have other ones like loot anime for all you fucking weebs out there.
Don't forget to like, subscribe, and click the bell so you're aware when I make more PRs in the future.
:cl:
* rscadd: Added loot crates. A monthly subscription for $30 you can buy in the cargo merch computer.

### CODE REVIEW

**Comment by DrSnips** [comment]
Slap that honk tag on this PR and lets get things started.

**Comment by Aurx** [comment]
Needs a sprite.

**Comment by DrSnips** [comment]
It's a child of the purple present so it uses that sprite.
edit: I might make a sprite for it later.

File: code/game/objects/items/weapons/gift_wrappaper.dm [hunk 1]
```diff
@@ -313,3 +313,14 @@
...
+
+///Loot crate///
+/obj/item/weapon/winter_gift/cloth/dorkcube
+	name = "dork cube"
+	desc = "Don't forget to shout the magic phrase when you open it!"
+
+/obj/item/weapon/winter_gift/cloth/dorkcube/attack_self(mob/user)
+	user.say("Loot get!")
+	playsound(get_turf(src), 'sound/misc/achievement.ogg', 30, 1)
+	user.gib()
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/gift_wrappaper.dm]: Should've saved it for April Fools and it might've actually gotten merged like this

**Comment by Aurx** [comment]
Yeah, that's what I meant.  Needs its own sprite.

**Comment by DrSnips** [comment]
ert box
The box sprite used for people who don't want to search for what the box looks like.

File: code/game/objects/items/weapons/gift_wrappaper.dm (hunk 1 above)
Comment (DrSnips) [reply code/game/objects/items/weapons/gift_wrappaper.dm]: If it doesn't go through now I can bring it back later by reopening the PR at that time.

**Comment by calzilla1** [comment]
Instead of it gibbing the opener, why not just go through with the meme and put a splash of blood on the ground and make the player bloody

**Comment by DrSnips** [comment]
You're so happy about your loot crate you just burst with happiness.

**Comment by SonixApache** [comment]
just lock it to a holiday or something

**Comment by Blithering** [comment]
Because tool is loot backwards
lootbox

**Comment by yclatious** [comment]
This meme just got better.

**Comment by small-boss** [comment]
THERES BLOOD IN THE BOX

**Comment by Exxion** [comment]
# ?

**Pull request merged**